packages/relay/src/lib/hbarlimiter/index.ts

```diff
@@ -52,6 +54,26 @@ export default class HbarLimit {
       registers: [register],
       labelNames: ['mode', 'methodName'],
     });
+    this.hbarLimitCounter.inc(0);
+
+    const rateLimiterRemainingGaugeName = 'rpc_relay_hbar_rate_remaining';
+    register.removeSingleMetric(rateLimiterRemainingGaugeName);
+    this.hbarLimitRemainingGauge = new Gauge({
+      name: rateLimiterRemainingGaugeName,
+      help: 'Relay Hbar limit total',
+      registers: [register],
+    });
+    this.hbarLimitRemainingGauge.set(this.remainingBudget);
+
+    const shouldLimitCounterName = 'rpc_relay_hbar_should_limit';
```

Comment:
I don't think this metric is as valuable but if you feel strongly you can split it out into a different ticket

Reply:
Done.